docs/docsite/rst/module_plugin_guide/plugin_filtering_config.rst

```diff
@@ -17,7 +17,7 @@ If you want to avoid using certain modules, you can add them to a reject list to
 
 The file contains two fields:
 
-  * A file version so that you can update the format while keeping backwards compatibility in the future. The present version should be the string, ``"1.0"``
+  * A file version so that you can update the format while keeping backward compatibility in the future. The present version should be the string, ``"1.0"``
```

Comment:
```suggestion
  * A file version so that you can update the format while keeping backwards compatibility in the future. The present version should be the string, ``"1.0"``
```